library/std/src/backtrace.rs

```diff
@@ -104,35 +104,103 @@ use crate::sync::Once;
 use crate::sys_common::backtrace::{lock, output_filename};
 use crate::vec::Vec;
 
+pub use core::backtrace::Backtrace;
+pub use core::backtrace::BacktraceStatus;
+use core::backtrace::RawBacktrace;
+
+/// Returns whether backtrace captures are enabled through environment
+/// variables.
+#[export_name = "__rust_backtrace_enabled"]
```

Comment:
```suggestion
#[export_name = "rust_backtrace_enabled"]
```
I don't think this should not use the `__` prefix. At least `rust_begin_unwind`, `rust_oom` and `rust_eh_personality` don't use it.

Reply:
Unfortunately this does not solve the compilation issues 😢

Reply:
> Unfortunately this does not solve the compilation issues cry

The compiler errors are just from the test suite, I wouldn't expect them to be super hard to fix.

Reply:
I ended up fixing this by using `#[lang ...]` instead of `#[export_name ...]` and now I have no idea why I even used `export_name` in the first place...